plugins/modules/cloud/opennebula/one_vm.py

```diff
@@ -752,11 +752,19 @@ def get_vm_info(client, vm):
     if 'NIC' in vm.TEMPLATE:
         if isinstance(vm.TEMPLATE['NIC'], list):
             for nic in vm.TEMPLATE['NIC']:
-                networks_info.append({'ip': nic['IP'], 'mac': nic['MAC'], 'name': nic['NETWORK'], 'security_groups': nic['SECURITY_GROUPS']})
+                networks_info.append({
+                    'ip': vm.TEMPLATE['NIC'].get('IP',''),
```

Comment:
shouldn't this be `nic.get('IP', '')` here?

Reply:
you are right, copy & paste eror